tests/system/Pager/PagerTest.php

```diff
@@ -255,4 +255,34 @@ public function testGetNextURIReturnsNullOnFirstPage()
 	}
 
 	//--------------------------------------------------------------------
+
+	public function testGetNextURIWithQueryStringUsesCurrentURI()
+	{
+		$_GET['page'] = 3;
```

Comment:
It's safer to completely reset the GET array during each test so it will overwrite anything that any other tests might try to use it for. So, something like: 

```
$_GET = [
    'page' => 3,
    'status' => 1
];
```

Or reset the GET variable to an empty array during teardown/setup. That keeps things clean for both this test file and others.